cirq/circuits/circuit_dag.py

```diff
@@ -38,6 +38,11 @@ def __init__(self, val: T) -> None:
     def __repr__(self):
         return 'Unique({}, {!r})'.format(id(self), self.val)
 
+    def __lt__(self, other):
```

Comment:
I think PEP8 likes you to implement all of these.  Maybe use functools.total_ordering()?

Reply:
Done.